locales/en-US/server.ftl

```diff
@@ -217,7 +218,8 @@ shotIndexPagePreviousPage
     .title = Previous page
 shotIndexPageNextPage
     .title = Next page
-
+shotIndexNoExpirationSymbol = ∞
```

Comment:
Can you add some context in a comment on where this symbol is used? In particular, make it clear that there is not space available for more text, if necessary it should be replaced by another single character like `-`.

Reply:
Good point.  Will do.